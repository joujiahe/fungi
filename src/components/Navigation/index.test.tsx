import { render } from "@testing-library/react";
import { Navigation } from "./index";

it("renders Navigation unchanged", () => {
  const { container } = render(<Navigation />);
  expect(container).toMatchSnapshot();
});
