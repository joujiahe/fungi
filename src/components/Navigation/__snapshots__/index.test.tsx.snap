// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Navigation unchanged 1`] = `
<div>
  <div
    class="css-1nsvf7n"
  >
    <a
      href="/"
    >
      <h2
        class="chakra-heading css-0"
      >
        fungi
      </h2>
    </a>
    <div
      class="css-1084q30"
    >
      <a
        class="chakra-link css-1isemmb"
        href="/collections"
      >
        Collections
      </a>
      <a
        class="chakra-link css-15fzge"
        href="/orders"
      >
        Orders
      </a>
    </div>
  </div>
</div>
`;
